core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/NotificationDAO.scala

```diff
@@ -1,32 +0,0 @@
-package org.broadinstitute.dsde.rawls.dataaccess
```

Comment:
For reviewers: almost all of the changes are removing the code duplicated by Rawls and pointing to the version in workbench-libs